libnavigation-core/src/test/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSessionTest.kt

```diff
@@ -109,82 +99,6 @@ class MapboxDirectionsSessionTest {
         verify { router.cancel() }
     }
 
-    @Test
-    fun fasterRoute_timerStartedOnce() {
-        session.requestRoutes(routeOptions, routesRequestCallback)
-        callback.onResponse(routes)
-        session.requestRoutes(routeOptions, routesRequestCallback)
-        verify(exactly = 1) { mapboxTimer.start() }
-    }
-
-    @Test
-    fun fasterRoute_timerShutdown() {
-        session.shutDownSession()
-        verify { mapboxTimer.stop() }
-    }
-
-    @Test
-    fun fasterRoute_hasRoute() {
-        session.requestRoutes(routeOptions, routesRequestCallback)
-        callback.onResponse(routes)
-        delayLambda()
-        verify(exactly = 1) { router.getRoute(routeOptions, callback) }
-    }
-
-    @Test
-    fun fasterRoute_noRoutes() {
-        delayLambda()
-        verify(exactly = 0) { router.getRoute(any(), any()) }
-    }
-
-    @Test
-    fun fasterRoute_failedRoutes() {
-        session.requestRoutes(routeOptions, routesRequestCallback)
-        val throwable: Throwable = mockk()
-        callback.onFailure(throwable)
-        delayLambda()
-        verify(exactly = 1) { router.getRoute(any(), any()) }
-    }
-
-    @Test
-    fun fasterRoute_inProgress() {
-        session.requestRoutes(routeOptions, routesRequestCallback)
-        delayLambda()
-        verify(exactly = 1) { router.getRoute(any(), any()) }
-    }
-
-    @Test
-    fun fasterRoute_canceled() {
-        session.requestRoutes(routeOptions, routesRequestCallback)
-        verify(exactly = 1) { router.cancel() }
-        callback.onResponse(routes)
-
-        clearMocks(router)
-        delayLambda()
-        session.cancel()
-        verify(exactly = 1) { router.cancel() }
-    }
-
-    @Test
-    fun fasterRoute_canceledByNewRequest() {
```

Comment:
We need to recreate this test - when a faster route request is sent, it should be canceled whenever a new, main route is requested/set.